docs/sources/installation/helm/reference.md

```diff
@@ -2207,6 +2207,15 @@ null
 			<td><pre lang="json">
 []
 </pre>
+</td>
```

Comment:
Note the comment up at line 9:
"Autogenerated. Modify ../production/helm/loki/reference.md.gotmpl "

Reply:
don't know what you mean I just ran
`make -C docs sources/installation/helm/reference.md`
as the CI job indicated